src/Aspire.Dashboard/Components/Pages/TraceDetail.razor.cs

```diff
@@ -166,7 +166,14 @@ private void UpdateDetailViewData()
 
     private string GetRowClass(SpanWaterfallViewModel viewModel)
     {
-        return (viewModel.Span.SpanId == _span?.SpanId) ? "selected-span" : string.Empty;
+        if (viewModel.Span == SelectedSpan?.Span)
+        {
+            return "selected-row";
+        }
+        else
+        {
+            return (viewModel.Span.SpanId == _span?.SpanId) ? "selected-span" : string.Empty;
```

Comment:
I don't think the concept of selected-span is still used. There used to be a link on the logs page to a span, and it would be selected. The logs link was removed.

But I can double check and clean up later.